README.md

```diff
@@ -77,6 +77,8 @@ Receive notifications of critical security updates. The sophistication of online
 
 - **Twitter** - Follow [Mattermost](https://twitter.com/mattermost)
 - **Blog** - Get the latest updates from the [Mattermost blog](https://mattermost.com/blog/).
+- **facebook** - Follow [Mattermost](https://www.facebook.com/MattermostHQ)
+- **linkedin** - Follow [Mattermost](https://www.linkedin.com/company/mattermost/)
```

Comment:
```suggestion
- **Facebook** - Follow [Mattermost](https://www.facebook.com/MattermostHQ)
- **Linkedin** - Follow [Mattermost](https://www.linkedin.com/company/mattermost/)
```

4/5 suggest we capitalize the first letters, to match the other listings.  cc @jasonblais what do you think?